docs/service_specification.schema.json

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "backstage.io/v1beta1",
```

Comment:
are these meant to be stable over time? or can this be like `backstage.io/catalog/entity`?

Reply:
`The $id keyword defines a URI for the schema, and the base URI that other URI references within the schema are resolved against.`

So I think probably this should point to something in backstage.io or maybe to the github url to the master branch for this file.